### PR TITLE
Fixed #24358 -- Corrected code-block directives for console sessions.

### PR DESCRIPTION
**Pull request by sww**
Switched the code blocks from bash to console in the Django
documentation and ensured the appropriate lines had a leading $.

### CODE REVIEW

File: docs/internals/contributing/writing-code/unit-tests.txt [hunk 1]
```diff
@@ -28,7 +28,7 @@ Running the tests requires a Django settings module that defines the
 databases to use. To make it easy to get started, Django provides and uses a
 sample settings module that uses the SQLite database. To run the tests:
 
-.. code-block:: bash
+.. code-block:: console
```
Comment (berkerpeksag) [inline docs/internals/contributing/writing-code/unit-tests.txt]: I'd add `.. highlight:: console` to top of the file then switch to use this style:

```rst
[...] To run the tests::  # notice two :

    $ git clone https://github.com/django/django.git django-repo
    cd django-repo/tests
```

Comment (MarkusH) [inline docs/internals/contributing/writing-code/unit-tests.txt]: Likely not. I expect Python to be default language and thus use `::` for that. If a file has a huge majority for another language, I'd be okay with setting the default highlight to that language.

File: docs/internals/howto-release-django.txt [hunk 2]
```diff
@@ -56,7 +56,7 @@ You'll need a few things before getting started:
 
 * An install of some required Python packages:
 
-  .. code-block:: bash
+  .. code-block:: console
```
Comment (MarkusH) [inline docs/internals/howto-release-django.txt]: I'd switch to the `.. highlight:: console` style in this file as well and then fix the current `::` code sections to be console sections (e.g. prepand `$ ` to the lines)

**Comment by sww** [comment]
Updated the changes; let me know if I did something wrong or if you have more suggestions!

File: docs/internals/contributing/writing-code/unit-tests.txt [hunk 3]
```diff
@@ -1,3 +1,5 @@
+.. highlight:: console
```
Comment (timgraham) [inline docs/internals/contributing/writing-code/unit-tests.txt]: I'd put this below the heading.

File: docs/internals/contributing/writing-code/unit-tests.txt [hunk 4]
```diff
@@ -276,26 +256,20 @@ will pair ``test_eq`` with every test label.
...
 
     $ ./runtests.py basic --reverse
 
 If you wish to examine the SQL being run in failing tests, you can turn on
 :ref:`SQL logging <django-db-logger>` using the ``--debug-sql`` option. If you
 combine this with ``--verbosity=2``, all SQL queries will be output.
-
-.. code-block:: bash
-
+::
```
Comment (timgraham) [inline docs/internals/contributing/writing-code/unit-tests.txt]: You can change this to `all SQL queries will be output::` for consistency.

**Comment by timgraham** [comment]
You can squash commits with those updates, thanks. Also, please make sure the "docs" build failure is addressed.

**Comment by timgraham** [comment]
Updated and merged in eba6dff581aa8bd6a1c08456e83e68ad09ae4ec3, thanks.

**Pull request closed without merging**